.env.default

```diff
@@ -16,8 +14,6 @@ IMAGE_SOLR=solr:8-slim
 IMAGE_REDIS=redis:5-alpine
 IMAGE_DRIVER=zenika/alpine-chrome
 CLEAR_FRONT_PACKAGES=no
-ADD_PHP_EXT=graphicsmagick php7-intl
-#ADD_PHP_EXT=php7-pdo_pgsql postgresql-client postgresql-contrib gnu-libiconv wkhtmltopdf php7-redis
```

Comment:
Please keep this line as it helpful to maintain a list of most-used packages

Reply:
This remains here https://github.com/skilld-labs/skilld-docker-container/pull/401/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52L44